metricbeat/docs/modules/aws.asciidoc

```diff
@@ -14,38 +14,16 @@ The default metricsets are `ec2`, `sqs`, `s3_request`, `s3_daily_storage`, `clou
 [float]
 === Module-specific configuration notes
 
-This module uses environment variable `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` and `AWS_REGION`
-that are referenced in the config file to set values that need to be configurable during deployment. It also
-accept optional configuration `regions` to specify what are the AWS regions to query metrics from. If `regions`
-parameter is not set in the config file, then by default, aws module will query metrics from all available
+`aws` module requires AWS credentials configuration in order to make AWS API calls.
+Users can either use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and/or
+`AWS_SESSION_TOKEN`, or use shared AWS credentials file.
+Please see <<aws-credentials-config>> for more details.
```

Comment:
@dedemorton I'm trying to reference/add link of one asciidoc in another asciidoc file. This `aws-credentials-config` is located at `x-pack/libbeat/docs/aws-credentials-config.asciidoc`. What's the correct way to link here?

Reply:
Hmmm I think I figured it out?! But if it doesn't look right, please let me know :-) Thanks!

Reply:
You have a couple of problems here.

1 - The build is not pulling in the included file because it's looking under `/elastic/libbeat/docs/aws-credentials-config.asciidoc`, but the file is actually located under `x-pack/libbeat/docs/aws-credentials-config.asciidoc`.

It's a best practice to use attributes to resolve include paths for libbeat docs. Now that we have a libbeat directory under x-pack, I'd recommend adding a new attribute to `metricbeat/docs/index.asciidoc`, so you would have: 

```
= Metricbeat Reference

:libbeat-dir: ../../libbeat

:libbeat-xpack-dir: ../../../x-pack/libbeat


include::{libbeat-dir}/docs/version.asciidoc[]
```

Then use the new attribute where you include the file:

```
include::{libbeat-xpack-dir}/docs/aws-credentials-config.asciidoc[]
```

This will fix the error about the included file.

2 - The structure of the included file needs to work with the content that you're pulling it into. I'll say more about that later.

Reply:
@dedemorton Thank you so much for all the help! The link works now!